packages/entities/entities-shared/src/composables/useExternalLinkCreator.ts

```diff
@@ -7,11 +7,12 @@ export default function useExternalLinkCreator(routeParams: string[]): string {
     const fullUrl = `${baseUrl}${path}`
     const hasEmptyParams = routeParams.some(item => item.trim() === '')
 
-    /* Test validity of generated URL */
-    const urlRegex = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,4}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g // eslint-disable-line no-useless-escape
+    // Test validity of generated URL; only allows `http` protocal if domain is `localhost`
+    /* eslint-disable-next-line no-useless-escape */
+    const validUrlRegex = /^(https:\/\/(www\.)?([a-zA-Z0-9-]+\.){1,}[a-zA-Z]{2,}(:[0-9]+)?(\/[^\/]*)*)$|^http:\/\/localhost(:[0-9]+)?(\/[^\/]*)*$/
```

Comment:
technically i think the last `[^\/]*` should probably be `+` if you want to avoid doubleslash?
```
> t = /^(\/[^\/]*)*$/
/^(\/[^\/]*)*$/
> t.test('/a//')
true
> t.test('//a')
true
> t = /^(\/[^\/]+)*$/
/^(\/[^\/]+)*$/
> t.test('//a')
false
> t.test('/a//')
false
```

Reply:
Yes, but we do want to allow a single slash, since we might want to land on `/gateway-manager/176e2ec5-d3c0-4268-ac25-d8716ed8c14f/routes` for example.

Reply:
Scratch that, we don't do trailing slashes. Pushed a commit w/ `[^\/]+`.